docs/src/modules/components/AppSearch.js

```diff
@@ -219,11 +251,19 @@ export default function AppSearch() {
         type="search"
         id="docsearch-input"
         inputRef={inputRef}
+        onFocus={() => {
+          setFocused(true);
+        }}
+        onBlur={() => {
+          setFocused(false);
+        }}
         classes={{
           root: classes.inputRoot,
           input: classes.inputInput,
         }}
       />
+      {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
+      <div className={clsx(classes.shortcut, { 'Mui-focused': focused })}>/</div>
```

Comment:
We should display the dual-key keyboard shortcut here instead so that it's more apparent what this is supposed to do. We're currently using the most exotic shortcut.

Reply:
We are leveraging what GitHub is teaching developers. I think that we better display it this way. Using / is a common convention.

Reply:
> Using / is a common convention.

Please explain how you define common.

Reply:
Common as a shortcut our audience has already learned, so they can keep previous habits. e.g. GitHub, Gmail, Google Calendar, DocSearch, Apollo docs, Twitter.

Reply:
How do you know that the audience has learned it? 

>  e.g. GitHub, Gmail, DocSearch, Twitter, Apollo docs.

Please make this exhaustive. I just want to be able to work with you in feature PRs. So far you change you references every time and it would be nice to know if these are actual principles or just smoke screens for "I just prefer it that way".

Reply:
I think that we have spent enough time on the topic, relative to its importance. I think that it's a detail. I think that solving this problem is enough: create awareness among developers familiar with the habit that Material-UI implements shortcuts on the search bar. For this, a short label is enough. I think that `CTRL + K` would create more visual distraction that it needs to. I think that we should ignore WCAG 2.1.4 because our target audience for the documentation is developers. From what I understand, your are also in favor of ignoring it by keeping the alises for `s` and `/`.

Reply:
- Storybook

<img width="202" alt="Capture d’écran 2020-12-14 à 19 34 54" src="https://user-images.githubusercontent.com/3165635/102120968-8c268e00-3e43-11eb-95dc-52b4a495449f.png">

- https://linear.app/

<img width="211" alt="Capture d’écran 2020-12-14 à 19 35 55" src="https://user-images.githubusercontent.com/3165635/102121244-f3dcd900-3e43-11eb-9568-8f7d55bfab2f.png">

Yet, they support `/` and `s`, and don't display any shortcut in the app.

Reply:
@eps1lon Considering it's a detail, that I think we are wasting time on this subject, and that we can probably solve my previous concern of visual distraction by reducing the contrast of the displayed shortcut, I think that if you think strongly about it, we can move forward with displaying Ctrl + K by default. 

I also think that supporting `/` and `s` is important. Maybe we could do solve https://github.com/mui-org/material-ui/pull/23959#discussion_r541948700 with 2. for these two shortcuts and support Ctrl + K broadly?

Reply:
> I think that we have spent enough time on the topic, relative to its importance.

You have no arguments anymore. 

> I also think that supporting / and s is important.

Why?

Reply:
> Why?

Because developers have learned this shortcut.

Reply:
> Developers have learned this shortcut

What's your evidence for that? Why does this not apply to ctrl+k?

Reply:
Ok, let's drop them and wait for developers to complain.